### ISSUE TITLE
Unexpected space removal in multiline comments

### ISSUE THREAD

**Issue by rousskov** [Mark-Simulacrum/attractifier#5]
This formatting change does not look attractive:

```
 /* Tests an HTTP proxy with a Headman eCAP adapter
- * configured to hide and reveal empty Content-Type headers. */
+ *configured to hide and reveal empty Content-Type headers. */
 
```

**Comment by Mark-Simulacrum** [comment]
Please confirm that this is fixed in the latest version (v1.5.1).

**Comment by rousskov** [comment]
Confirmed.
